apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexOrderProcessor.java

```diff
@@ -63,13 +73,17 @@
     private IDexValidator dexValidator;
     private DexSmartContractService dexSmartContractService;
     private EthereumWalletService ethereumWalletService;
+    private TaskDispatchManager taskDispatchManager;
+
+    private volatile boolean processorEnabled = true;
+    @Getter
+    private boolean initialized = false;
```

Comment:
Seems like this field is not using

Reply:
I see: 
 @PostConstruct
    public void init(){.... initialized = true;